From foo@baz.org Fri Nov 18 09:11:43 2005
Date: Fri, 18 Nov 2005 22:18:50 +0700
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: update Documentation/00-INDEX

Update the index file with descriptions of the stable_api_nonsense.txt
and stable_kernel_rules.txt files.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/00-INDEX |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/Documentation/00-INDEX
+++ gregkh-2.6/Documentation/00-INDEX
@@ -258,6 +258,10 @@ specialix.txt
 	- info on hardware/driver for specialix IO8+ multiport serial card.
 spinlocks.txt
 	- info on using spinlocks to provide exclusive access in kernel.
+stable_api_nonsense.txt
+	- info on why the kernel does not have a stable in-kernel api or abi.
+stable_kernel_rules.txt
+	- rules and procedures for the -stable kernel releases.
 stallion.txt
 	- info on using the Stallion multiport serial driver.
 svga.txt
